include/caffe/filler.hpp

```diff
@@ -94,6 +94,59 @@ class GaussianFiller : public Filler<Dtype> {
   shared_ptr<SyncedMemory> rand_vec_;
 };
 
+
+/// @brief Fills a Blob with Gaussian-distributed values @f$ x = a @f$.
+//         RNG is only called once, and its output is stored in
+//         GaussianStaticFillerdata. Followup calls to the Filler function
+//         will bypass RNG.
+template <typename Dtype>
+class GaussianStaticFiller : public Filler<Dtype> {
+ public:
+  explicit GaussianStaticFiller(const FillerParameter& param)
+      : Filler<Dtype>(param) {
+        GaussianStaticFillerFlag=false;
+      }
+  virtual void Fill(Blob<Dtype>* blob) {
+    Dtype* data = blob->mutable_cpu_data();
+    CHECK(blob->count());
```

Comment:
You use `blob->count()` seven times in this code. Why not to create a variable? Kinda
`const int len = blob->count();`